idea/explorer/src/decorators/method.ts

```diff
@@ -76,5 +85,38 @@ export function JsonRpc<TBase extends Constructor<JsonRpcBase>>(Base: TBase) {
         };
       }
     }
+
+    createRestRouter(): Router {
+      const router = Router();
+
+      for (const { method, path, handler } of restHandlers) {
+        router[method](path, async (req, res) => {
+          const { genesis } = req.query;
```

Comment:
Perhaps it's better to have `genesis` in the body of the request as is done in jsonrpc requests